tools/deb/istio-iptables.sh

```diff
@@ -157,6 +181,41 @@ if [ -z "${PROXY_GID}" ]; then
 PROXY_GID=${PROXY_UID}
 fi
 
+#
+# Since OUTBOUND_IP_RANGES_EXCLUDE could carry ipv4 and ipv6 ranges
+# need to split them in different arrays one for ipv4 and one for ipv6
+# in order to not to fail
+pl='/*'
+ipv6_ranges_exclude=()
+ipv4_ranges_exclude=()
+for range in "${OUTBOUND_IP_RANGES_EXCLUDE[@]}"; do
```

Comment:
General bash comment:  I think this is the first use in this script of bash arrays.  I think there might be bash version portability problems with those.

Reply:
This code runs in istio_init container and we do control which image is used. So chances of hitting compatibility issue imho is very low.

Reply:
Well in k8s env it does, but this script is also used in mesh expansion VMs and we have a mirror version currently in Istio CNI that runs in the host OS.  I'll leave for others to comment, e.g. @costinm 

Would be good to move to go as you mentioned in slack but that's longer term.